### PR TITLE
Spatial agent tweaks

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

They get 100 armor armor and nodrop glasses

## Why It's Good For The Game

Stop stealing the glasses

suit is at sunos request
## Changelog
:cl:
add: Spatial agent suit
tweak: Spatial agent glasses are nodrop
/:cl:

### CODE REVIEW

**Comment by Pariah919** [comment]
Making spatial agents glasses nodrop is fine, giving them a dumb suit that makes them look like an enemy is not and is horrifically stupid. You have mass sleep if you don't want to get shot by marines for whatever reason.

File: code/modules/clothing/glasses/glasses.dm [hunk 1]
```diff
@@ -279,6 +279,10 @@
 	. = ..()
 	AddComponent(/datum/component/clothing_tint, TINT_NONE)
 
+/obj/item/clothing/glasses/sunglasses/sa/nodrop
+	desc = "Glasses worn by a spatial agent. cannot be dropped"
+	flags_item = NODROP|DELONDROP
```
Comment (psykzz) [inline code/modules/clothing/glasses/glasses.dm:284]:
```suggestion
	flags_item = DELONDROP
```

**Comment by jroinc** [comment]
But why?

Why do you have to be invincible and untouchable in the mode you assumed to directly interact with people?

**Comment by psykzz** [comment]
Glasses are stupid and should be nodrop the rest i agree, if you want to ahelp yourself you can, or just respawn as SA again.

**Comment by Hathkar** [comment]
The outfit is a bit much, and having the glasses delete on drop would be better, in my opinion. As a spatial agent I sometimes need to take the glasses off to confirm vision related things.

**Pull request merged**